h2o-algos/src/main/java/hex/word2vec/Word2VecMojoWriter.java

```diff
@@ -0,0 +1,45 @@
+package hex.word2vec;
+
+import hex.ModelMojoWriter;
+import water.MemoryManager;
+import water.parser.BufferedString;
+
+import java.io.IOException;
+import java.nio.ByteBuffer;
+
+/**
+ * MOJO serializer for word2vec model.
+ */
+public class Word2VecMojoWriter extends ModelMojoWriter<Word2VecModel, Word2VecModel.Word2VecParameters, Word2VecModel.Word2VecOutput> {
+
+  @SuppressWarnings("unused")  // Called through reflection in ModelBuildersHandler
+  public Word2VecMojoWriter() {}
+
+  public Word2VecMojoWriter(Word2VecModel model) {
+    super(model);
+  }
+
+  @Override public String mojoVersion() {
+    return "1.00";
+  }
+
+  @Override
+  protected void writeModelData() throws IOException {
+    writekv("vec_size", model._parms._vec_size);
+    writekv("vocab_size", model._output._words.length);
+
+    // Vocabulary
+    startWritingTextFile("vocabulary");
+    for (BufferedString word : model._output._words) {
+      writeln(word.toString()); // FIXME: escape new lines
```

Comment:
@st-pasha do we have a convenient function for escaping of new line characters?

string can already contain "\n", eg.:

"this\none
string"

should be escaped to "this\\none\nstring"